hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/cluster/strategy/PartitionAwareClusteringPlanStrategy.java

```diff
@@ -113,6 +114,15 @@ public PartitionAwareClusteringPlanStrategy(HoodieTable table, HoodieEngineConte
         .build());
   }
 
+  public List<String> getMatchedPartitions(HoodieWriteConfig config, List<String> partitionPaths) {
+    String partitionSelected = config.getClusteringPartitionSelected();
+    if (!StringUtils.isNullOrEmpty(partitionSelected)) {
+      return Arrays.asList(partitionSelected.split(","));
```

Comment:
`delimiter`which is comma by default?

Reply:
Yes, the filtered partition is separated by comma when pruning partition, see the end of `org.apache.spark.sql.hudi.command.procedures.RunClusteringProcedure#prunePartition` for more detail.